prog/vm/nexus.rb

```diff
@@ -94,7 +94,7 @@ def self.assemble(public_key, project_id, name: nil, size: "standard-2",
       end
 
```

Comment:
Although somewhat repetitive, the commit message

 >  Materialize Vm's memory_gib and vcpus.  

Should be expanded. It's chancy and annoying to navigate from a blame in the hope that the previous commits explain why this was done (though in this case, it does). 

An alternative: is hyperlink via commit hash, which is super awkward given rebasing. May work well if you merge the migration (giving it a permanent hash) and then edit this. That would be the most terse, though.

Reply:
Expanded the commit message.